src/utils/naming.ts

```diff
@@ -5,34 +5,44 @@ export interface DomainCapitalizedOptions {
   domainPascalCase: string;
 }
 
-interface SubStoreCapitalizedOptions {
+export interface SubStoreCapitalizedOptions {
   subStoreKebabCase: string;
   subStoreAllCaps: string;
   subStoreCamelCase: string;
   subStorePascalCase: string;
 }
 
 export const kebabify = (str: string): string =>
-  str.replace('_', '-').replace(/[A-Z]+(?![a-z])|[A-Z]/g, ($, ofs) => (ofs ? '-' : '') + $.toLowerCase());
-export const camelize = (str: string) => str.replace(/-./g, (x) => x[1].toUpperCase());
+  str
+    .replace("_", "-")
+    .replace(
+      /[A-Z]+(?![a-z])|[A-Z]/g,
+      ($, ofs) => (ofs ? "-" : "") + $.toLowerCase()
+    );
+export const camelize = (str: string) =>
+  str.replace(/-./g, (x) => x[1].toUpperCase());
 
 export function formatCapitalizations(str: string, isDomain: boolean) {
-  const dashToUnderScores = str.replace('-', '_');
+  const dashToUnderScores = str.replace("-", "_");
```

Comment:
🤔 Maybe we should copy the .prettierrc and pre-commit hooks over to nx-angular-generators ?

Reply:
Yes i thought about that too.... Also we should consider running prettier on all files in one big PR in both repos.
https://github.com/trifork/nx-angular-generators/issues/13